config/config.default.cson

```diff
@@ -49,6 +49,7 @@
         "state": "public.posts"
         "params": {}
     "exposeAsideWhen": "(min-width:900px)"
+    "logo": "/img/logo.png" # null will disable
```

Comment:
You need to use the path to the config file `../config/logo.png` (not sure about the relative path but something similar :))